src/System.Reflection/tests/AssemblyTests.cs

```diff
@@ -332,6 +333,13 @@ public void LoadFile()
             {
                 Assert.Equal(loadedAssembly1, loadedAssembly2);
             }
+
+            AssemblyLoadContext alc = AssemblyLoadContext.GetLoadContext(loadedAssembly1);
+            string expectedName = string.Format("Assembly.LoadFile({0})", fullRuntimeTestsPath);
```

Comment:
Since `fullRuntimeTestsPath` gets updated between loading `loadedAssembly1` and `loadedAssembly2`, shouldn't this be the expected name for the ALC with `loadedAssembly2` instead of `loadedAssembly1`?
Or is the point that they should both have the same name since they had the same effective path? (If so, comment?)

Reply:
Actually it was wrong for both.  The second load hit the cache and returned the same one created in the first call. So I needed the unmodified path.